watertap/examples/flowsheets/oaro/oaro_multi_ui.py

```diff
@@ -0,0 +1,667 @@
+        exports.add(
+            obj=pump.efficiency_pump[0],
+            name=f"Primary pump {idx} efficiency",
+            ui_units=pyunits.dimensionless,
+            display_units="fraction",
+            rounding=2,
+            description=f"Efficiency of primary pump {idx}",
+            is_input=True,
+            input_category="Primary Pumps",
+            is_output=False,
+        )
+
+    # Unit model data, recycle pumps
+    for idx, pump in fs.RecyclePumps.items():
+        exports.add(
+            obj=pump.efficiency_pump[0],
+            name=f"Recycle pump {idx} efficiency",
+            ui_units=pyunits.dimensionless,
+            display_units="fraction",
+            rounding=2,
+            description=f"Efficiency of recycle pump {idx}",
+            is_input=True,
+            input_category="Recycle Pumps",
+            is_output=False,
+        )
+
+    # Unit model data, ERD
+    for idx, erd in fs.EnergyRecoveryDevices.items():
+        exports.add(
+            obj=erd.efficiency_pump[0],
+            name=f"ERD {idx} pump efficiency",
+            ui_units=pyunits.dimensionless,
+            display_units="fraction",
+            rounding=2,
+            description=f"Efficiency of energy recovery device {idx}",
+            is_input=True,
+            input_category="Energy Recovery Device",
+            is_output=False,
+        )
+        exports.add(
+            obj=erd.control_volume.properties_out[0].pressure,
+            name=f"ERD {idx} operating pressure",
+            ui_units=pyunits.bar,
+            display_units="bar",
+            rounding=2,
+            description=f"Operating pressure of energy recovery device {idx}",
+            is_input=True,
+            input_category="Energy Recovery Device",
+            is_output=False,
+        )
+
+    # System costing
+    exports.add(
+        obj=fs.costing.utilization_factor,
+        name="Utilization factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TIC,
+        name="Total Installed Cost",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Total Installed Cost (TIC)",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TPEC,
+        name="Total Purchased Equipment Cost",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Total Purchased Equipment Cost (TPEC)",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.factor_total_investment,
```

Comment:
@luohezhiming This should be `total_investment_factor`

Reply:
I think it should be factor_total_investment, please check `watertap_costing_package.py`

Reply:
I push the modification, hope there is no other conflicts